channeldb/error.go

```diff
@@ -1,33 +1,39 @@
 package channeldb
 
-import "fmt"
+import (
+	"errors"
+)
 
 var (
-	ErrNoChanDBExists    = fmt.Errorf("channel db has not yet been created")
-	ErrLinkNodesNotFound = fmt.Errorf("no link nodes exist")
+	ErrNoChanDBExists    = errors.New("channel db has not yet been created")
+	ErrLinkNodesNotFound = errors.New("no link nodes exist")
+
+	ErrNoActiveChannels = errors.New("no active channels exist")
+	ErrChannelNoExist   = errors.New("this channel does not exist")
+	ErrNoPastDeltas     = errors.New("channel has no recorded deltas")
 
-	ErrNoActiveChannels = fmt.Errorf("no active channels exist")
-	ErrChannelNoExist   = fmt.Errorf("this channel does not exist")
-	ErrNoPastDeltas     = fmt.Errorf("channel has no recorded deltas")
+	ErrInvoiceNotFound   = errors.New("unable to locate invoice")
+	ErrNoInvoicesCreated = errors.New("there are no existing invoices")
+	ErrDuplicateInvoice  = errors.New("invoice with payment hash already exists")
 
-	ErrInvoiceNotFound   = fmt.Errorf("unable to locate invoice")
-	ErrNoInvoicesCreated = fmt.Errorf("there are no existing invoices")
-	ErrDuplicateInvoice  = fmt.Errorf("invoice with payment hash already exists")
+	ErrNoPaymentsCreated = errors.New("there are no existing payments")
 
-	ErrNoPaymentsCreated = fmt.Errorf("there are no existing payments")
+	ErrNodeNotFound = errors.New("link node with target identity not found")
+	ErrMetaNotFound = errors.New("unable to locate meta information")
 
-	ErrNodeNotFound = fmt.Errorf("link node with target identity not found")
-	ErrMetaNotFound = fmt.Errorf("unable to locate meta information")
+	ErrGraphNotFound      = errors.New("graph bucket not initialized")
+	ErrGraphNodesNotFound = errors.New("no graph nodes exist")
+	ErrGraphNoEdgesFound  = errors.New("no graph edges exist")
```

Comment:
The diff in this file should be made minimal: don't re-organize the order of any of the other variables, simply append the new error types to the end of the declaration block.